app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::ItemsController < ActionController::API
```

Comment:
At the moment inheriting from action_controller::API is required for spec harness tests to pass